### PR TITLE
[PW-7778] - Implement GooglePay as an individual payment method

### PR DESCRIPTION
**Pull request by Morerice**
**Description**
This PR will add GooglePay as an individual payment method. Order will be have payment method set to `adyen_googlepay`.

**Tested scenarios**
* Successful payment
* Webhook processed
* GooglePay window closed during checkout

### CODE REVIEW

File: view/frontend/web/js/view/payment/method-renderer/adyen-paypal-method.js [hunk 1]
```diff
@@ -76,6 +76,10 @@ define(
                         street: formattedShippingAddress.street
                     };
                 }
+
+                // Paypal requires extra configuration
```
Comment (candemiralp) [inline view/frontend/web/js/view/payment/method-renderer/adyen-paypal-method.js:80]: Is this related to the GooglePay?

File: Model/Methods/GooglePay.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+ */
+namespace Adyen\Payment\Model\Methods;
+
+use Adyen\Payment\Helper\PaymentMethods\PaymentMethodInterface;
+use Adyen\Payment\Model\AdyenPaymentMethod;
+
+class GooglePay extends AdyenPaymentMethod implements PaymentMethodInterface
+{
+    const CODE = 'adyen_googlepay';
+    const TX_VARIANT = 'googlepay';
```
Comment (candemiralp) [inline Model/Methods/GooglePay.php:20]: Is there a way to support `paywithgoogle`?

File: view/frontend/web/js/view/payment/method-renderer/adyen-paypal-method.js (hunk 1 above)
Comment (Morerice) [reply view/frontend/web/js/view/payment/method-renderer/adyen-paypal-method.js:80]: This was a fix required for PayPal that I also included in this PR

File: Model/Methods/GooglePay.php (hunk 2 above)
Comment (Morerice) [reply Model/Methods/GooglePay.php:20]: We could somehow make it work, but since we are already asking merchants to use `googlepay` from v8 upwards, I don't think it's necessary

**Review by candemiralp — APPROVED** [review]

**Review by RokPopov — APPROVED** [review]
